OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -354,7 +354,7 @@ public interface IMove
 
 	public interface IRadarSignature
 	{
-		IEnumerable<Pair<CPos, Color>> RadarSignatureCells(Actor self);
+		void RadarSignatureCells(Actor self, List<Pair<CPos, Color>> destinationBuffer);
```

Comment:
Considering that a cell can have only one colour, would it make sense to change `List<Pair<CPos, Color>>` to a `Dictionary<CPos, Color>`?

Reply:
IMO it would be clearer if we changed the method name to something like `PopulateRadarSignatureCells` to go with the signature change.  Does anybody else have an opinion on this?

Reply:
A dictionary would cause issues for the editor, which does everything in one shot and thus can have multiple cells in the same space. I think the suggested name is reasonable and have applied it.

Reply:
The minimap can only render one color, so wouldn't that be an optimization?
We'd just need to assign using the indexer rather than .Add.

Reply:
We'd also incur the overhead of a dictionary, and since we're not doing any lookups, that would seem to be a poor tradeoff.

Reply:
Alright, fair enough.